### ISSUE TITLE
High level players are not being properly credited the right amount of faction reputation

### ISSUE THREAD

**Issue by Nelythia** [elysium-project/server#2846]
If you are above the level range for a quest that you complete, you do not get the full reputation reward.

For example if you complete a quest that gives you +75 reputation during its level range, you get +75 reputation. 
If you in fact complete it outside of the level range, lets say as level 60, you don't get +75, you get around 10 or 20 reputation.

This is a blizzard confirmed issue which was patched with the release of TBC
http://wowwiki.wikia.com/wiki/Patch_2.0.1

'Fixed an issue where high level players were not being properly credited the right amount of faction reputation.'

**Comment by Oudeis1447** [comment]
Yes, you said it, TBC. I wouldn`t call it a bug either, it was clearly made on purpose and they decide to change it in TBC. This is not a TBC server.

**Comment by Nelythia** [comment]
it is a bug acknowledged by Blizzard, they just haven't fixed it until TBC.
TBC has been used as evidence previously here. For example how quests are scripted, how mobs interact with players, how spells work. All has had some evidence and influence of TBC changes.
